static/app/views/dashboardsV2/dashboard.tsx

```diff
@@ -44,7 +47,7 @@ const ROW_HEIGHT = 120;
 const WIDGET_MARGINS: [number, number] = [16, 16];
 const ADD_BUTTON_POSITION = {
   x: 0,
-  y: Number.MAX_VALUE,
+  y: Number.MAX_SAFE_INTEGER,
```

Comment:
Change it to this (9007199254740991) instead of `Number.MAX_VALUE` since django doesn't like 1.7976931348623157e+308